web/packages/teleterm/src/ui/services/workspacesService/documentsService/types.ts

```diff
@@ -134,6 +134,11 @@ export interface DocumentGatewayCliClient extends DocumentBase {
   status: '' | 'connecting' | 'connected' | 'error';
 }
 
+/**
+ * DocumentGatewayKube replaced DocumentTshKube in Connect v14. Before removing DocumentTshKube
```

Comment:
does "document" in `DocumentXXX` mean like "tabs" ? i know we have some document naming too in teleport but the meaning has always been elusive to me

Reply:
Yep, I think this is where the teleterm package got the concept of documents from. The way I think about it is that a Document is the equivalent of `window.document` in a way, meaning that it represent a particular page under that tab.